From bdc2940825d8eb827bc3136a822e10ad201ba453 Mon Sep 17 00:00:00 2001
From: Eric Van Hensbergen <ericvh@kernel.org>
Date: Sat, 6 May 2023 16:34:41 +0000
Subject: [PATCH] Weird cast error seems to cause trouble

This seems to generative invalid argument errors
in my setup, not sure why this doesn't show up elsewhere.

Signed-off-by: Eric Van Hensbergen <ericvh@kernel.org>
---
 client/cpu_linux.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/cpu_linux.go b/client/cpu_linux.go
index f2ed9d8..214ae6f 100644
--- a/client/cpu_linux.go
+++ b/client/cpu_linux.go
@@ -12,7 +12,7 @@ import (
 )
 
 func osflags(fi os.FileInfo, mode p9.OpenFlags) int {
-	flags := int(mode)
+	flags := mode.OSFlags()
 	if fi.IsDir() {
 		flags |= syscall.O_DIRECTORY
 	}
-- 
2.34.1

